src/modules/previewpane/powerpreview/settings.cpp

```diff
@@ -0,0 +1,145 @@
+#include "pch.h"
+#include <common.h>
+#include "settings.h"
+#include "trace.h"
+
+namespace PowerPreviewSettings
+{
+	extern "C" IMAGE_DOS_HEADER __ImageBase;
+
+	// Base Settinngs Class Implementation
+	FileExplorerPreviewSettings::FileExplorerPreviewSettings(bool state, std::wstring name, std::wstring description)
+	{
+        this->m_isPreviewEnabled = std::move(state);
```

Comment:
what is the purpose of `std::move`?

Reply:
https://github.com/microsoft/PowerToys/pull/1075#discussion_r369763912

Reply:
thanks this makes sense :)